SPHINXsys/src/for_3D_build/particles/solid_particles_supplementary.cpp

```diff
@@ -42,21 +51,47 @@ namespace SPH {
 		return 1.0/(1.0 + poisson) * std::sqrt(0.5 * (std::pow(eps_1 - eps_2, 2) + std::pow(eps_2 - eps_3, 2) + std::pow(eps_3 - eps_1, 2)));
 	}
```

Comment:
instead of std::pow, you can use the powerN function from scalar_functions.h which will be computationally more efficient.

Reply:
done, thanks!